renovate.json

```diff
@@ -1,49 +1,39 @@
 {
-  "extends": [
-    "config:base",
-    ":disableDependencyDashboard",
-    ":preserveSemverRanges"
-  ],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:node"],
   "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": [
-    "after 9am every weekday",
-    "before 5pm every weekday"
-  ],
-  "enabledManagers": ["npm", "nvm", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
   "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "rangeStrategy": "bump",
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major NPM dependencies",
+      "groupSlug": "all-npm-minor-patch",
+      "stabilityDays": 3
```

Comment:
As per [Renovate docs](https://docs.renovatebot.com/configuration-options/#rangestrategy)

> bump = e.g. bump the range even if the new version satisfies the existing range, e.g. ^1.0.0 -> ^1.1.0

This should avoid us needing to manually bump minor/patch dependencies manually

Then have introduced `stabilityDays`

> Number of days required before a new release is considered stable.

and `groupName` - to group them together in a single PR once they are at least 3 days old - this should ensure we aren't continually swamped with PRs. I spent some time testing this config on https://github.com/ministryofjustice/renovate-typescript-poc/pulls but can see how we find it?